src/FINDOLOGIC/Request/Parameters/ParameterBuilder.php

```diff
@@ -61,18 +61,24 @@ public function addReferer($value)
      *      you are not sure which value you should pass to the API.
      * @see https://docs.findologic.com/doku.php?id=integration_documentation:request#required_parameters
      */
-    public function addRevision($value)
+    public function setRevision($value)
     {
         $revision = new Revision();
         $revision->setValue($value);
         $this->addParam(Revision::PARAM_KEY, $value);
```

Comment:
Returning `$this` in all builder methods could enable chaining by creating a fluent interface.